src/compiler.jl

```diff
@@ -357,17 +357,21 @@ end
 end
 
 @inline @generated function active_reg(::Type{T}) where {T}
+    JuliaContext() do ctx
```

Comment:
Why do we need JuliaContext here?
Furthermore alignment.

Reply:
Not needed here and now removed. Also removed in a few other places and only exists in trampolines, thunk, deferred_codegen and enzyme_call. The do bodies are now indented.